pulp_rpm/app/viewsets.py

```diff
@@ -299,7 +299,7 @@ def _process_config(self, config):
             r["dest_repo"] = dest_repo.pk
             repos.extend((source_version.repository, dest_repo))
 
-            if entry.get("dest_base_version"):
```

Comment:
When ```dest_base_version``` == 0, it functions as though this method were returning ```None``` :(

The perils of dynamic typing.